### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by shhdharmen** [ngneat/hot-toast#11]
## I'm submitting a...

- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [x] Bug report  
- [ ] Performance issue
- [ ] Feature request
- [ ] Documentation issue or request
- [ ] Support request
- [ ] Other... Please describe:

### Accessibility Issues

1. Icon does not have `aria-hidden="true"`

### Accessibility Features

1. Icon should have `aria-hidden="true"`

### Doc

Also some documentation regarding Accessibility is needed

## What is the motivation / use case for changing the behavior?

Hot toasts should be accessible by default.

**Comment by shhdharmen** [comment]
https://github.com/ngneat/hot-toast/commit/42793155ebbe8da6744616b79ed5e16e774e1d1c

**Comment by shhdharmen** [comment]
https://github.com/ngneat/hot-toast/commit/b3e5691d1df18f020bce08684109c0dcf0398d45
